addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -3,257 +3,234 @@
-            style="grid-area: 1 / 4 / 11 / 7; z-index: 2;">
-            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item o_grid_item_image g-col-lg-3 rounded g-height-4 col-lg-3" data-name="Block" style="z-index: 2; grid-area: 1 / 4 / 5 / 7;">
+            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto rounded" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 7 / 11 / 10; z-index: 3;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc2 g-col-lg-3 rounded g-height-4 col-lg-3 justify-content-start" data-name="Block" style="z-index: 3; grid-area: 1 / 7 / 5 / 10; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px;">
+            <h3 class="h4-fs">Advanced <br class="d-none d-lg-inline"/>Capabilities</h3>
+            <p>&#160;</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 10 / 11 / 13; z-index: 4;">
-            <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item o_grid_item_image g-col-lg-3 rounded g-height-4 col-lg-3" data-name="Block" style="z-index: 4; grid-area: 1 / 10 / 5 / 13;">
+            <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto rounded" alt=""/>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_alternation_image_text_template" groups="base.group_user">
     <div class="row o_grid_mode" data-row-count="10">
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 1 / 11 / 4; z-index: 1;">
+        <div class="o_grid_item o_grid_item_image col-lg-3 g-col-lg-3 g-height-10 justify-content-center text-center" data-name="Block" style="grid-area: 1 / 1 / 11 / 4; z-index: 1;">
             <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 1 / 4 / 11 / 7; z-index: 2;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 justify-content-center text-center o_cc o_cc2" data-name="Block" style="grid-area: 1 / 4 / 11 / 7; z-index: 2; position: relative;" data-oe-shape-data="{'shape':'web_editor/Floats/05','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0','animated':'true'}">
```

Comment:
- The shape name should be `Rainy/06`, like you defined it below :)

Reply:
good catch 👍